File: programmers/python 3/70129.py
# 문제 링크: https://programmers.co.kr/learn/courses/30/lessons/70129

def solution(s):
    counts = [0, 0]
    while s != '1':
        zeros = s.count('0')
        s = bin(len(s) - zeros)[2:]
        counts[0] += 1
        counts[1] += zeros
    return counts

File: programmers/swift/70129.swift
// 문제 링크: https://programmers.co.kr/learn/courses/30/lessons/70129

func solution(_ s: String) -> [Int] {
    var s = s
    var counts = [0, 0]

    while s != "1" {
        let nonzeros = s.filter { $0 == "1" }.count
        let zeros = s.count - nonzeros

        s = String(nonzeros, radix: 2)
        counts[0] += 1
        counts[1] += zeros
    }

    return counts
}